src/category_theory/discrete_category.lean

```diff
@@ -96,6 +96,24 @@ begin
   simp,
 end
 
+omit 𝒞
+
+variables {A : Type*} {B : Type*} (f : A ≃ B)
+
+def equiv_unit_iso : 𝟭 (discrete A) ≅ lift (f.to_fun) ⋙ lift (f.inv_fun) :=
```

Comment:
Wouldn't it be better to use a type ascription to coerce `f` to a function, instead of using the projection? Otherwise you might prevent simp-lemmas from triggering...